UnoCpp/UnoCpp/GameManager.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include <memory>
+#include <vector>
+
+class Player;
+class BoardController;
+
+class GameManager
+{
+public:
+    std::shared_ptr<std::vector<Player>> Players;
```

Comment:
to na dúvida quanto a isso. Pra mim parece mais correto se for `std::vector<std::shared_ptr<Player>> Players;`. Me parece mais razoável eu ter um objeto vector com ponteiros pra Player do que um ponteiro pra um vector de Players.